fem/gridfunc.hpp

```diff
@@ -306,6 +306,29 @@ class GridFunction : public Vector
                                  Coefficient *weight = NULL,
                                  const IntegrationRule *irs[] = NULL) const;
 
+   /** Compute the Lp error in each element of the mesh and store the
+       results in a GridFunction, error.  The result should be an L2
+       GridFunction of order zero using map type VALUE. */
+   virtual void ComputeLpError(const double p, Coefficient &exsol,
```

Comment:
I think a better name for this method is `ComputeElementLpErrors`. The same applies to the `L1`, `L2`, and `Max` versions below, as well as the `VectorCoefficient` versions.